scripts/ovadownload.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
Please add a comment explaining what this script is used for.

Reply:
Script removed.